packages/dev/create-react-app/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@parcel/create-react-app",
```

Comment:
AFAICT, `packages/dev` is for packages that aren't published, so it should rather be `packages/utils`?